key-import-export/key_exchange/README.md

```diff
@@ -6,6 +6,14 @@ KDH : Key Distribution Host
 KRD : Key Receiving Device
 Futurex : HSM is configured using PMK
 
+## Configure AWS credentials
```

Comment:
AWS credentials are already needed. Add a section on why roles need to be configured and what use case it supports.